packages/bitcore-node/src/config.ts

```diff
@@ -1,38 +1,35 @@
-import { cpus, homedir } from 'os';
+import fs from 'fs';
+import { cpus } from 'os';
+import logger from './logger';
 import { ConfigType } from './types/Config';
 import { merge } from './utils';
-import parseArgv from './utils/parseArgv';
 
-const program = parseArgv([], ['config']);
+const bitcoreConfigPath = process.env.BITCORE_CONFIG || '../../bitcore.config.json';
 
-function findConfig(): ConfigType | undefined {
-  let foundConfig;
-  const envConfigPath = process.env.BITCORE_CONFIG_PATH;
-  const argConfigPath = program.config;
-  const configFileName = 'bitcore.config.json';
-  const bitcoreConfigPaths = [
-    `${homedir()}/${configFileName}`,
-    `../../../../${configFileName}`,
-    `../../${configFileName}`
-  ];
-  const overrideConfig = argConfigPath || envConfigPath;
-  if (overrideConfig) {
-    bitcoreConfigPaths.unshift(overrideConfig);
-  }
-  // No config specified. Search home, bitcore and cur directory
-  for (const path of bitcoreConfigPaths) {
-    if (!foundConfig) {
-      try {
-        const expanded = path[0] === '~' ? path.replace('~', homedir()) : path;
-        // eslint-disable-next-line @typescript-eslint/no-require-imports
-        const bitcoreConfig = require(expanded) as { bitcoreNode: ConfigType };
-        foundConfig = bitcoreConfig.bitcoreNode;
-      } catch {
-        foundConfig = undefined;
-      }
-    }
-  }
-  return foundConfig;
+logger.info('Using config at: ' + bitcoreConfigPath);
+
+if (!fs.existsSync(bitcoreConfigPath)) {
+  throw new Error(`No bitcore config exists at ${bitcoreConfigPath}`);
+}
+
+const bitcoreConfigStat = fs.statSync(bitcoreConfigPath);
+
+if (bitcoreConfigStat.isDirectory()) {
+  throw new Error(`Provided bitcore config path: ${bitcoreConfigPath} is a directory`);
```

Comment:
If `bitcoreConfigPath` is a dir, it'd be better to append `bitcore.config.json` (`path.join(bitcoreConfigPath, 'bitcore.config.json'`) and try that. So in pseudocode:

```
if dir {
  path = path + 'bitcore.config.json'
}

if !exists {
  throw error
}
```